packages/shared/routes/dashboard/staking/views/StakingInfo.svelte

```diff
@@ -16,8 +13,22 @@
     import { selectedAccountId } from '@lib/wallet'
 
     let animation: string
+    let header: string
+    let subHeader: string
 
-    const updateAnimation = (): void => {
+    $: isAssemblyStaked = $selectedAccountParticipationOverview?.assemblyStakedFunds > 0
+    $: isShimmerStaked = $selectedAccountParticipationOverview?.shimmerStakedFunds > 0
+    $: $selectedAccountParticipationOverview, $stakingEventState, setHeaders()
+    $: isAssemblyStaked, isShimmerStaked, $stakingEventState, $selectedAccountId, setAnimation()
+
+    enum FileNumber {
```

Comment:
What does this name mean exactly?

Reply:
It's the number of the animation to be shown. Should it be changed to `AnimationFileNumber` or `AnimationNumber`?